config/routes.rb

```diff
@@ -4,6 +4,7 @@
 
   devise_for :users, :controllers => { registrations: 'registrations' }
 
+  get 'tasks/:id/markdown', to: 'tasks#markdown_help_model', as: "markdown_modal"
```

Comment:
Maybe move to another controller? Or maybe there's a gem that will help with such routes that actually are not part of some MVC.